commands.js

```diff
@@ -1481,6 +1481,8 @@ var commands = exports.commands = {
 			return;
 		}
 		room.destroyLog(function () {
+			clearInterval(Users.backupLocksBans);
```

Comment:
no

(I can't think of a way `/kill` could fail, but if it does, PS should clearly keep backing up locks and bans)